File: qlty-cli/tests/lang/java/basic.in/fields/ClassDeclaration.java
package fields;

class Foo {
  String bar;
  String baz;

  // Constructor
  public Foo() {
    this.bar = "";
    this.baz = "";
  }
}

public class ClassDeclaration {
  public static void main(String[] args) {
    System.out.println(doSomething());
  }

  public static String doSomething() {
    Foo foo = new Foo();
    foo.bar = "Hello";
    foo.baz = "World";
    return foo.bar + foo.baz;
  }
}

File: qlty-cli/tests/lang/csharp/basic.in/fields/ClassDeclaration.cs
namespace Fields
{
    class Foo
    {
        public string bar;
        public string baz;

        // Constructor
        public Foo()
        {
            this.bar = "";
            this.baz = "";
        }
    }

    public class ClassDeclaration
    {
        public static void Main(string[] args)
        {
            System.Console.WriteLine(DoSomething());
        }

        public static string DoSomething()
        {
            Foo foo = new Foo();
            foo.bar = "Hello";
            foo.baz = "World";
            return foo.bar + foo.baz;
        }
    }
}